tests/providers/amazon/aws/operators/test_sagemaker_transform.py

```diff
@@ -43,6 +43,16 @@
     "TransformOutput": {"S3OutputPath": "output_path"},
     "TransformResources": {"InstanceType": "ml.m4.xlarge", "InstanceCount": "3"},
 }
+CREATE_TRANSFORM_PARAMS_INTEGER_FIELDS: dict = {
+    "TransformJobName": "job_name",
+    "ModelName": "model_name",
+    "MaxConcurrentTransforms": 12,
+    "MaxPayloadInMB": 6,
+    "BatchStrategy": "MultiRecord",
+    "TransformInput": {"DataSource": {"S3DataSource": {"S3DataType": "S3Prefix", "S3Uri": "s3_uri"}}},
+    "TransformOutput": {"S3OutputPath": "output_path"},
+    "TransformResources": {"InstanceType": "ml.m4.xlarge", "InstanceCount": 3},
+}
```

Comment:
Note: previous implementation relied on the fact that executing the operator actively modified the `CREATE_TRANSFORM_PARAMS` constant, which is bad practice. Here I provide a second dictionary with the expected integer values and modified tests accordingly.